classes/submission.class.php

```diff
@@ -1371,8 +1384,8 @@ private function reset_retries() {
     /**
      * Common method to update maximum retry attempts and error Status.
      *
-     * @param $error string Error code from Turnitin.
-     * @param $retry_attempts int Maximum retry attempts.
+     * @param string $error Error code from Turnitin.
```

Comment:
The parameter type needs to be after the variable name otherwise you'll get warnings about the doc comments.

Reply:
Actually, if we keep it after a variable name then it will give warning.

Reply:
Yeah you're right! There's a few I added in my PR that are incorrect as I thought it was after the name. Looks like I need to run the doc checker on mine as well.